internal/kafka/internal/services/quota/ams_quota_service.go

```diff
@@ -167,7 +177,15 @@ func (q amsQuotaService) ValidateBillingAccount(organisationId string, instanceT
 	return nil
 }
 
-func (q amsQuotaService) CheckIfQuotaIsDefinedForInstanceType(username string, externalId string, instanceType types.KafkaInstanceType) (bool, *errors.ServiceError) {
+func (q amsQuotaService) CheckIfQuotaIsDefinedForInstanceType(username string, externalId string, instanceType types.KafkaInstanceType, billingModel services.KafkaBillingModel) (bool, *errors.ServiceError) {
```

Comment:
the `billingModel` parameter has been added to the `CheckIfQuotaIsDefinedForInstanceType` method. Should this be removed from here too for now in this PR? (https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1368/files#r1015433365) or we want to keep it so it is there already?

Reply:
I see now it is due to the interface it implements has that field added too and it is used by the quota management list implementation. Let's add a comment in here anyway as I understand this implementation will need to be changed too